app/src/main/java/nl/tudelft/b_b_w/blockchain/Hash.java

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.b_b_w.blockchain;
+
+/**
+ * Class for creating a Hash object
+ */
+
+public class Hash {
+
+    /**
+     * Properties of a hash
+     */
+    private String hashString;
+
+    /**
+     * Constructor for Hash class
+     * @param hash given a hash String
+     */
+    public Hash(String hash) {
+        this.hashString = hash;
+    }
+
+    /**
+     * toString() getter implementation
+     * @return String representation of the hash
+     */
+    @Override
+    public String toString() {
```

Comment:
I would say Hash is only a String holder, sounds redundant, but I think it is better to make a Hash class that represents the hash itself instead of using String all the time. I think this is a good practice.